libs/component-sample/src/lib/feature/side-navigation-sample/side-navigation-sample.component.html

```diff
@@ -2,17 +2,20 @@
     <sds-page-options class="sam-page__options">
         <sds-toolbar [expanded]="sideNavExpanded" #toolbar="sdsToolbar">
             <sds-accordion multi="true">
-                <sds-accordion-item [expanded]="sideNavExpanded">
+                <sds-accordion-item #navigationAccordion [expanded]="navigationExpanded">
                     <sds-accordion-item-header>
                         <div class="sds-accordion__item-header-title">Navigation</div>
-                        <div class="sds-accordion__item-header-subtitle">Selected Category</div>
+                        <div class="sds-accordion__item-header-subtitle">{{ pageHeader }}</div>
                     </sds-accordion-item-header>
-                    <sds-side-navigation #sideNav (linkEvent)="linkEvent.next($event)" [model]="model">
+                    <sds-side-navigation #sideNav  [model]="model">
                     </sds-side-navigation>
                 </sds-accordion-item>
-                <sds-accordion-item>
+                <sds-accordion-item #filtersAccordion>
                     <sds-accordion-item-header> Filters </sds-accordion-item-header>
-                    <sds-filters [form]="form" [fields]="fields" [model]="model1" (filterChange)="filterChange$.next($event)"></sds-filters>
+                    <sds-filters [form]="form" [fields]="fields" [model]="model1" (filterChange)="filterChange$.next($event)">
```

Comment:
filterChange is unnecessary/doesn't do anything

Reply:
I can remove this line and instances of filter change, but this isn't code that I wrote.